wagtail/images/models.py

```diff
@@ -298,6 +298,11 @@ class AbstractImage(ImageFileMixin, CollectionMember, index.Indexed, models.Mode
 
     objects = ImageQuerySet.as_manager()
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.is_decorative = False
```

Comment:
Fixed in a7d99dadbef7015ec5f1cf251072c32ddc24c581 to use `decorative` consistently. (Before my additions there was just the one isolated use of `is_decorative` in `__init__`, which was harmless since a missing `decorative` attribute is equivalent to `decorative = False` everywhere else - I ended up propagating that typo into `__eq__` / `__hash__`...)